deploy/linux/roles/prepare/tasks/installSupervisord.yml

```diff
@@ -22,7 +22,7 @@
     when: pip_installed.stdout|int == 1 and pip_local.stdout|int == 0
   
   - name: Installing latest pip - fetch
-    shell: curl -sSL https://bootstrap.pypa.io/2.7/get-pip.py -o get-pip.py
+    shell: curl -sSL https://bootstrap.pypa.io/pip/2.7/get-pip.py -o get-pip.py
```

Comment:
i thought this change was already made. might be missing a rebase or merge w/ main.

Reply:
Good catch, definitely forgot to pull that in